src/middlewared/middlewared/plugins/system.py

```diff
@@ -820,6 +820,18 @@ def _get_index(country_columns, column):
 
         return verrors
 
+    @accepts()
+    async def ui_certificate_choices(self):
+        """
+        Return choices of `ui_certificate` attribute for `system.general.update`.
```

Comment:
```suggestion
        Return `system.general.update` `ui_certificate` attribute choices.
```

Reply:
Appreciate the suggestion but we are pass on this one if you dont mind. Consulted with other people and they agreed original version is just as much readable.

Thanks!